topics/ui/controls/tooltip.md

```diff
@@ -10,40 +10,46 @@
 
 </tldr>
 
-A tooltip shows an action name or provides useful information about an action or a setting.
+A tooltip appears on hovering over a UI element and shows an action name or provides useful information about an action or a setting.
 
-![](01_top_pic.png){width=174}
+<img src="tooltip.png" alt="Tooltip" width="706"/>
 
 ## When to use
 
 There are three types of tooltips:
 
-<table style="none">
+<table border="false" column-width="fixed" style="none">
   <tr>
-    <td width="50%">
+    <td>
+        <img src="tooltip_action.png" alt="Action tooltip" />
+    </td>
+    <td>
         <strong>Action </strong>
         <p>Shows an action name or label for icons and unlabeled controls, and a shortcut if available.</p>
-    </td>
-    <td width="50%">
-        <img src="02_regular_tooltip.png" width="218" />
+        <p><control>Required</control>: action name</p>
+        <p><control>Optional</control>: shortcut</p>
```

Comment:
I'd suggest removing bold styling for required/optional because it makes less clear that the bold "Action", "Action help" and "Help" are actually subheaders